application/values.yaml

```diff
@@ -182,7 +182,7 @@ deployment:
     exec: {}
 
   readinessProbe:
-    enabled: true
+    enabled: false
```

Comment:
These should be carefully considered based on the purpose of this chart which is to enforce best practices not as a way to have a 3 line deployment

Best practice is that all deployments should have probes and the current config forces a developer to define their probes or explicitly disable them rather than defaulting to no probes